Source/VCGeneration/Split.cs

```diff
@@ -1481,8 +1481,10 @@ private static Cmd AssertIntoAssume(Cmd c)
 
       public void ReleaseChecker()
       {
-        Checker.GoBackToIdle();
-        checker = null;
+        if (checker != null) {
```

Comment:
Can you use `checker?.GoBackToIdle()` ?

Reply:
It would just do `this.checker = null;` which I think isn't strictly necessary but I'd keep it in for now. I'm guessing `Split` shouldn't have a checker field but that's how it is for now.

Reply:
`this.checker` is never assigned in the above snippet, would it be somewhere else?

Reply:
I'm not sure I understand your question. `Split.checker` is currently assigned a value in `Split.BeginCheck` and let's keep it that way. I've added a definition of `ResetChecker` in the above snippet to be more clear.

Reply:
Oh ok, that should work given this definition of `ResetChecker`